plugins/processors/awsapplicationsignals/internal/normalizer/attributesnormalizer.go

```diff
@@ -40,7 +44,6 @@ var attributesRenamingForMetric = map[string]string{
 	attr.AWSRemoteTarget:             common.MetricAttributeRemoteResourceIdentifier,
 	attr.AWSRemoteResourceIdentifier: common.MetricAttributeRemoteResourceIdentifier,
 	attr.AWSRemoteResourceType:       common.MetricAttributeRemoteResourceType,
-	attr.AWSRemoteDbUser:             common.MetricAttributeRemoteDbUser,
```

Comment:
`attributesRenamingForMetric` is created to maintain the list of attributes to be renamed for attributes in EMF logs, so keeping `AttributeRemoteDbUser` here is ok. But now I see how much confusion it caused from the current naming.

Reply:
In that case, I'll close this pull request if it's fine with you?

Reply:
Yes. Thank you!